### PR TITLE
remove unregistration in the legacy system

### PR DESCRIPTION
**Pull request by alexduf**

### CODE REVIEW

**Comment by NathanielBennett** [comment]
:+1: Looks good. but i think there's a couple more tests you should blat

File: registration/test/registration/controllers/MainControllerSpec.scala [hunk 1]
```diff
@@ -34,18 +34,6 @@ class MainControllerSpec extends PlaySpecification with JsonMatchers with Mockit
...
-      status(result) must equalTo(OK)
-      eventually(there was one(wsClient).url("https://localhost/device/registrations/gia:0E980097-59FD-4047-B609-366C6D5BB1B3"))
-    }
-
-    "accepts legacy android registration and unregisters from pushy" in new RegistrationsContext {
-      val Some(result) = route(app, FakeRequest(POST, "/legacy/device/register").withJsonBody(Json.parse(legacyAndroidRegistrationJson)))
-      status(result) must equalTo(OK)
-      eventually(there was one(wsClient).url("https://localhost/device/registrations/0e980097-59fd-4047-b609-366c6d5bb1b3"))
-    }
-
```
Comment (NathanielBennett) [inline registration/test/registration/controllers/MainControllerSpec.scala]: Can you not get rid of this test (L49) and also ""not include in invalid topics in response to legacy registration"  (L57 in master)

Comment (alexduf) [reply registration/test/registration/controllers/MainControllerSpec.scala]: good shout

Comment (alexduf) [reply registration/test/registration/controllers/MainControllerSpec.scala]: Actually, it looks unrelated to the unregistration going to the legacy system. Let's talk about it

Comment (NathanielBennett) [reply registration/test/registration/controllers/MainControllerSpec.scala]: Ok we've talked

Comment (NathanielBennett) [reply registration/test/registration/controllers/MainControllerSpec.scala]: :

**Pull request merged**